src/client/stylesheets/variables/_colours.scss

```diff
@@ -5,3 +5,10 @@ $btms-chart-blue: #5694ca;
 $btms-chart-green: #2ba8a3;
 $btms-chart-orange: #f57c00;
 $btms-chart-pink: #d81b60;
+
+$btms-support-line-numbers: #505a5f;
```

Comment:
we may as well use the same naming convention here as else where and change the prefix to:
`$btms-admin-*`